### DISCUSSION TITLE
lab1：table.cpp中first_page_id_有什么用

### DISCUSSION THREAD

**Discussion by guozhihao456** [thu-db/huadb#14] [Q&A]
请问table.cpp中的first_page_id_怎么理解，初始化的时候是NULL_PAGE_ID，之后在插入数据的时候需要变化么，这个first_page_id_有什么用

**Comment by Mai-icy** [comment]
first_page_id为NULL_PAGE_ID相当于没有页。
如果没有页，在插入数据后可以将first_page_id_设为0，之后的页面以链表形式连接，它相当于一个链表头的存在
